tests/h/search/query_test.py

```diff
@@ -513,11 +513,32 @@ def search(self, search):
 
 
 class TestUsersAggregation(object):
+    def test_it_aggregates_annotations_on_users(self, Annotation, search):
+        for i in range(2):
+            Annotation(userid="acct:pa@example.com")
+        Annotation(userid="acct:pb@example.com")
 
-    @pytest.fixture
-    def search(self, search):
         search.append_aggregation(query.UsersAggregation())
-        return search
+        result = search.run({})
+
+        count_for_pa = next(r for r in result.aggregations.get('users') if r["user"] == "acct:pa@example.com")['count']
```

Comment:
Can you please keep line lengths to < 100 chars, with <~ 90 recommended for most lines. Lines longer than become annoying to read in various tools (eg. GitHub's diff viewer).

Reply:
In this case depending on the complexity of the `result.aggregations` data structure it may simply be better to test for an exact match against a dict literal.